examples/docs_snippets/docs_snippets/concepts/assets/build_job.py

```diff
@@ -7,8 +7,8 @@ def asset1():
     return [1, 2, 3]
 
 
-@asset
-def asset2(asset1):
+@asset(deps=[asset1])
+def asset2():
     return asset1 + [4]
```

Comment:
re: the buildkit error - since `asset1` is listed as a `dep` there isn't a value that gets loaded. So in the old version of the example, the symbol `asset1` was the array `[1, 2, 3]`, but now it is the literal asset definition object. And you can't append a 4 to the asset definition, so that's where the error is coming from

Reply:
on that note, `asset1` should be updated so that it doesn't return `[1, 2, 3, 4]` since that would require the I/O manager